cpp/density/CorrelationFunction.cc

```diff
@@ -32,14 +32,13 @@ CorrelationFunction<T>::CorrelationFunction(unsigned int bins, float r_max) : Bo
     // We must construct two separate histograms, one for the counts and one
     // for the actual correlation function. The counts are used to normalize
     // the correlation function.
-    util::Histogram<unsigned int>::Axes axes;
+    util::Axes axes;
     axes.push_back(std::make_shared<util::RegularAxis>(bins, 0, r_max));
```

Comment:
Same suggestion as below -- use the `std::initializer_list<T>` form of the vector constructor so that this can be `const`.
```suggestion
    auto const axes = util::Axes {std::make_shared<util::RegularAxis>(bins, 0, r_max)};
```